### PR TITLE
report healthy organs on health scanner

### PR DESCRIPTION
**Pull request by ynot01**
# General Documentation

tested with zombie organ and sling organ on felinids

### Intent of your Pull Request

healthy organs will be reported on health scanners
antag organs will not show up unless damaged

### Why is this change good for the game?

doctors can now tell if someone is missing an organ

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

healthy organs will be reported on health scanners

### What should players be aware of when it comes to the changes your PR is implementing?

doctors can now tell if someone is missing an organ

### What general grouping does this PR fall under? 
Medical

# Changelog

:cl:  
rscadd: Added healthy organs to health scanner report
/:cl:

### CODE REVIEW

**Comment by Tipy1802** [comment]
> doctors can now tell if someone is missing an organ

They already get told X's y is missing
Seems like it would lead to information overload

**Comment by ynot01** [comment]
> > doctors can now tell if someone is missing an organ
> 
> They already get told X's y is missing
> Seems like it would lead to information overload

no they don't

**Comment by Tipy1802** [comment]
Ah my bad, it does that only for brain and heart when deffibing apperantly.
Yeah fair enough then, I would rather it tell you the missing organs instead of the present ones as it would make the health analyser into an even bigger wall of text but this is fine I guess

**Review by JamieD1 — APPROVED** [review]
